jenkins/data-prepper/release-data-prepper-all-artifacts.jenkinsfile

```diff
@@ -25,6 +25,19 @@ pipeline {
                 }
             }
         }
+        stage('Download Maven Artifacts') {
+            steps {
+                script {
+                    mavenPath = "${DATA_PREPPER_ARTIFACT_STAGING_SITE}/${VERSION}/${DATA_PREPPER_BUILD_NUMBER}/maven"
+                    group = "org/opensearch/dataprepper"
+                    artifacts = ["data-prepper-api"]
+                    fileTypes = ["-javadoc.jar", ".jar", ".pom", "-sources.jar", ".module"]
```

Comment:
Shouldn't the `jar` cover `-javadoc.jar` and `sources.jar`?

Reply:
We are building the filename from it right? We can't ls the list of files
`${artifact}-${VERSION}${fileType}${checksum}`
So, for example, in data-prepper-api-1.3.0-javadoc.jar `filetype` is `-javadoc.jar` and checksum is empty String.

Reply:
Makes sense. Thanks!